Word-to-PDF-Conversion/Change-track-changes-color/.NET/Change-track-changes-color/Change-track-changes-color.csproj

```diff
@@ -10,4 +10,18 @@
     <PackageReference Include="Syncfusion.DocIORenderer.Net.Core" Version="*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Data\" />
```

Comment:
Please remove this folder include code changes, by default folder will be added.

Reply:
Anna, removed the folder inclusion code.